src/Symfony/Component/Security/Core/User/ChainUserChecker.php

```diff
@@ -27,10 +29,16 @@ public function checkPreAuth(UserInterface $user): void
         }
     }
 
-    public function checkPostAuth(UserInterface $user): void
+    public function checkPostAuth(UserInterface $user /*, TokenInterface $token*/): void
     {
+        $token = 1 < \func_num_args() ? func_get_arg(1) : 1;
```

Comment:
Why does it fallback on 1 instead of something like `null` (even if I see that the future argument cannot be null) or false? It would make more sense to me than assigning an integer, no?

Reply:
That was a silly copy-and-paste mistake on my end. 🙈